ci_tools/bumpversion.sh

```diff
@@ -1,5 +1,16 @@
-bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
-git config --local user.email "engineering@finbourne.com"
-git config --local user.name "finbourne-bot-public"
-git push https://finbourne-bot-public:${gh_token}@github.com/finbourne/lusid-python-tools.git HEAD:$TRAVIS_BRANCH
-echo "Making commit with incremented version"
\ No newline at end of file
+#!/bin/bash
+
+last_commit_message=$(git log -1 HEAD --pretty=format:%s)
+
+echo "Last commit message: $last_commit_message"
```

Comment:
this looks like debug, I don't think we need to print it out

Reply:
Sure, removed.